### PR TITLE
add fluidvids.js v2.4.1 with git autoupdate

### PR DESCRIPTION
**Pull request by maruilian11**

### CODE REVIEW

File: ajax/libs/fluidvids.js/package.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  "author": "@toddmotto",
+  "license": "MIT",
+  "main": "dist/fluidvids.js",
+  "homepage": "https://github.com/toddmotto/fluidvids",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/toddmotto/fluidvids.git",
+    "basePath": "dist",
+    "files": [
+      "**"
```
Comment (PeterDaveHello) [inline ajax/libs/fluidvids.js/package.json]: Should be `**/*` not `**`.

**Comment by maruilian11** [comment]
fixed

**Comment by PeterDaveHello** [comment]
Can you update commit message to some kind of this:
`add fluidvids.js v2.4.1 with git autoupdate, close #5036`

**Comment by maruilian11** [comment]
ok

**Comment by maruilian11** [comment]
ok

**Pull request merged**